_posts/2021-08-23-rocket-validator.md

```diff
@@ -0,0 +1,113 @@
+---
+author: Jaime Iniesta
+author_link: https://jaimeiniesta.com
```

Comment:
```suggestion
author_link: https://github.com/jaimeiniesta
```

I believe the author link is used to power the author thumbnail image which pulls from your Github profile. For example, if you look at the post previews [here](https://beta.elixirschool.com/blog), the one missing an image is [another blog post](https://github.com/elixirschool/elixirschool/blob/master/_posts/2021-07-14-elixir-book-club.md) that uses a personal website link